async_firebase/messages.py

```diff
@@ -30,6 +30,13 @@ class AndroidNotification:
     title_loc_key: key of the title string in the app's string resources to use to localize the title text (optional).
     title_loc_args: a list of resource keys that will be used in place of the format specifiers in ``title_loc_key``
         (optional).
+    channel_id: The notification's channel id (new in Android O). The app must create a channel with this channel ID
```

Comment:
also add missing description of channel_id field according to https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#androidnotification

Reply:
Nice catch and thanks for fixing it.